packages/loot-core/src/client/query-helpers.ts

```diff
@@ -45,7 +45,7 @@ export function pagedQuery<TResponse = unknown>(
   );
 }
 
-type Data<TResponse> = ReadonlyArray<TResponse>;
+type Data<TResponse> = TResponse[];
```

Comment:
Updating all the downstream interfaces to be readonly would be a real big pain.. so hopefully you folks don't mind this change.